static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -43,6 +44,7 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
     const organization = useOrganization();
     const isOpen = useIsSelectedFeedback({feedbackItem});
     const hasReplayId = useFeedbackHasReplayId({feedbackId: feedbackItem.id});
+    const crashReportId = feedbackItem.metadata.source === 'crash_report_embed_form';
```

Comment:
Ah this isn't related to ids at all right?

```suggestion
    const isCrashReport = feedbackItem.metadata.source === 'crash_report_embed_form';
```

Reply:
oh yep forgot about this!